db/migrations/006.do.daily-desktop-users.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE daily_desktop_users (
+    day DATE NOT NULL,
+    platform TEXT NOT NULL,
+    user_count INT NOT NULL,
```

Comment:
INT has a max value of 2,147,483,647. I think that should be fine for desktop user count for the foreseeable future